tock/employees/admin.py

```diff
@@ -65,7 +65,7 @@ class UserDataAdmin(admin.ModelAdmin):
     search_fields = ('user__username',)
 
     def unit_info(self, obj):
-        return obj.get_unit_display()
+        return obj.unit.name
```

Comment:
This needs to check for `None` object.  It is currently breaking the admin interface.

Reply:
Nice catch Amy! Updated, and added a comment.